.github/workflows/pre-commit.yaml

```diff
@@ -1,14 +1,51 @@
 name: pre-commit
-
 on:
-  pull_request:
   push:
-    branches: [main]
+    branches-ignore:
```

Comment:
sorry why would you not run pre-commit on main?

Reply:
This is another `precommit::use_precommit(ci = "gha")` default

Reply:
I think this one has a case for not being run vs some of the others (as nothing should get to main without a PR. that being said it happens)

Reply:
I would think even in a hotfix we'd PR and force override branch protection? I'm not sure it's worth gaming this situation out too much because it's pretty extreme.

Reply:
Fixed.